### PR TITLE
Refer to the DNS HealthCheckSpeck from dns-operator repo

### PR DESCRIPTION
**Pull request by makslion**
Stop redefining DNSRecord HealthCheckSpec from dns-operator repo. 
Also, push to a fresher commit of the `dns-operator` so we have the latest CRD

### CODE REVIEW

File: bundle/manifests/kuadrant.io_dnspolicies.yaml [hunk 1]
```diff
@@ -69,37 +69,17 @@ spec:
               healthCheck:
                 description: |-
                   HealthCheckSpec configures health checks in the DNS provider.
-                  By default, this health check will be applied to each unique DNS A Record for
+                  By default this health check will be applied to each unique DNS A Record for
                   the listeners assigned to the target gateway
                 properties:
                   endpoint:
-                    description: |-
```
Comment (philbrookes) [inline bundle/manifests/kuadrant.io_dnspolicies.yaml:76]: it doesn't seem great to have lost the field descriptions, can they be re-added to the Spec in the DNS Operator to keep them?

Comment (makslion) [reply bundle/manifests/kuadrant.io_dnspolicies.yaml:76]: added those to the DNS Operator bu we will see them here only after the dns-operator PR gets merged

**Comment by makslion** [comment]
WIP until [this](https://github.com/Kuadrant/dns-operator/pull/160) one merges - will require a rebase

File: bundle/manifests/kuadrant.io_dnspolicies.yaml (hunk 1 above)
Comment (makslion) [reply bundle/manifests/kuadrant.io_dnspolicies.yaml:76]: @philbrookes not should be fine

**Review by mikenairn — APPROVED** [review]
Looks fine to me, maybe @philbrookes could take a quick look since its health check related.

**Review by philbrookes — APPROVED** [review]

**Pull request merged**